app/models/institution_builder.rb

```diff
@@ -383,7 +383,7 @@ def self.add_hcm(version_number)
     str = <<-SQL
       UPDATE institutions SET
         caution_flag = TRUE,
-        caution_flag_reason = concat_ws(',', caution_flag_reason, hcm_list.reasons)
+        caution_flag_reason = concat_ws(', ', caution_flag_reason, hcm_list.reasons)
```

Comment:
There is a spec in place for this that adds an accreditation caution first, and then an HCM. It uses a regex to match, which is why I missed the ',' rather than ', '. However, I did add a concatenation test for Accreditation - although it affects the caution flag reason first, if moved to another position in the build, the caution flag reason will still be concatenated properly.